xml/containers-get.xml

```diff
@@ -525,51 +497,45 @@
    </listitem>
    <listitem>
     <para>
-     pip packages
+     PyPI packages
     </para>
    </listitem>
    <listitem>
     <para>
-     npm packages
+     NPM packages
     </para>
    </listitem>
   </itemizedlist>
 
   <para>
-   These analyses can be performed on a single image, or a
-   <literal>diff</literal> can be performed on two images to compare.
+   You can inspect a single image, or perform a diff operation on two images.
```

Comment:
```suggestion
   You can inspect a single image, or perform a <literal>diff</literal> operation on two images.
```

Reply:
Alternatively, as "diff" isn't a word, you could expand it, e.g.

"perform a differential comparison of two images."